test/unit/math/src/mathComplexFloat.cpp

```diff
@@ -34,3 +35,17 @@ TEMPLATE_LIST_TEST_CASE("mathOpsComplexFloat", "[math] [operator]", TestAccFunct
     auto testTemplate = TestTemplate<Acc, Functor>{};
     testTemplate.template operator()<alpaka::Complex<float>>();
 }
+
+#ifdef __has_include
+#    if __has_include(<version>)
+#        include <version>
+#    endif
+#endif
```

Comment:
`__cpp_lib_is_layout_compatible` is also defined inside `<type_traits>`, so you can just include that header instead, which you will need anyway for `std::is_layout_compatible_v`.